ansible/roles_ocp_workloads/ocp4_workload_example/readme.adoc

```diff
@@ -22,25 +22,23 @@
 *** This role doesn't do anything here
```

Comment:
Why two readmes? Can we delete the other one?

Reply:
Deleting the other.  It's what's created by galaxy